projects/plugins/jetpack/extensions/blocks/blogroll/block.json

```
@@ -63,5 +56,11 @@
 			"type": "boolean",
 			"default": true
 		}
+	},
+	"providesContext": {
+		"showAvatar": "show_avatar",
+		"showDescription": "show_description",
+		"showSubscribeButton": "show_subscribe_button",
+		"openLinksNewWindow": "open_links_new_window"
```

If I'm not mistaken, only two are used

```suggestion
		"showSubscribeButton": "show_subscribe_button",
		"openLinksNewWindow": "open_links_new_window"
```